ProcessLib/SmallDeformation/SmallDeformationFEM.h

```diff
@@ -288,6 +293,74 @@ class SmallDeformationLocalAssembler
         }
     }
 
+    void readIntegrationPointData(std::vector<char> const& data) override
+    {
+        SmallDeformationFEM::ElementData element_data;
+        if (!element_data.ParseFromArray(data.data(), data.size()))
+            OGS_FATAL("Parsing ElementData protobuf failed.");
+
+        // check element number
+        if (_element.getID() != element_data.id())
+            OGS_FATAL(
+                "Reading input failed somewhat. Mesh item id does not match");
+
+        // check number of integration points
+        unsigned const n_integration_points =
+            _integration_method.getNumberOfPoints();
+        if (n_integration_points != element_data.n_integration_points())
+            OGS_FATAL(
+                "Reading input failed somewhat. The value of "
+                "n_integration_points does not match");
+
+        assert(n_integration_points == element_data.sigma_size());
+        for (unsigned ip = 0; ip < n_integration_points; ip++)
+        {
+            auto sigma = element_data.sigma(ip);
+            if (DisplacementDim != sigma.dimension())
+                OGS_FATAL("Dimension of a Kelvin vector do not match.");
+            assert(_ip_data[ip]._sigma.size() == sigma.value_size());
+
+            for (int i = 0; i < _ip_data[ip]._sigma.size(); ++i)
+                _ip_data[ip]._sigma[i] = sigma.value(i);
+        }
+    }
+
+
+    // Output data format:
+    //  mesh_item_id,
+    //  n = number_of_ipoints,
+    //  sigma_0, sigma_1...sigma_n.
```

Comment:
Removing.
This are just scary leftovers from sizeofreinterpretcastcharptr times. Documentation is in the .proto file (which is self explanatory ;)